cmd/definitions/gen_service.go

```diff
@@ -486,12 +486,18 @@ If user enable this feature, service should ignore not support pair error.`),
 				gg.Defer(gg.Embed(func() gg.Node {
 					caller := gg.Call("formatError").WithOwner("s")
 					caller.AddParameter(gg.Lit(fn.Name)).AddParameter("err")
+					paramsNum := 2
```

Comment:
It looks like we only need a `isEmpty := true` here, do we really need to count the nums?

Reply:
No, the num is unnecessary.